site/content/docs/5.1/components/offcanvas.md

```diff
@@ -27,8 +27,20 @@ Offcanvas is a sidebar component that can be toggled via JavaScript to appear fr
 
 Below is an offcanvas example that is shown by default (via `.show` on `.offcanvas`). Offcanvas includes support for a header with a close button and an optional body class for some initial `padding`. We suggest that you include offcanvas headers with dismiss actions whenever possible, or provide an explicit dismiss action.
 
-{{< example class="bd-example-offcanvas p-0 bg-light overflow-hidden" >}}
-<div class="offcanvas offcanvas-start" tabindex="-1" id="offcanvas" aria-labelledby="offcanvasLabel">
```

Comment:
if you add `.show` class here, instead of rewrite the example snippet, is not fine?

Reply:
Adding `.show` to the example as-is would bring up the backdrop over the docs pages. Maybe we could do that and disable the backdrop and scrolling?

`data-bs-scroll="true" data-bs-backdrop="false"`

Reply:
Tried this, but the autofocus on the dismiss button scrolls the page then. 🤔  Thoughts?

Reply:
You could try to add in _site/assets/js/application.js_ something like:
```js
// Offcanvas demo
 var myOffcanvas = document.querySelector('.bd-example-offcanvas #offcanvas');
 if (myOffcanvas) {
   myOffcanvas.addEventListener('show.bs.offcanvas', function(event){
     event.preventDefault();
   }, false);
 }
```
The advantage of keeping the `{{< example >}}` (if possible) is that we will continue to have the "Try it" button to go to StackBlitz.